VERSION

```diff
@@ -1 +1 @@
-4.0.3
\ No newline at end of file
+4.0.3
```

Comment:
nit: can you add back the newline so that there is no diff here

Reply:
Aliright.